lua/lean/infoview.lua

```diff
@@ -22,25 +29,36 @@ local _DEFAULT_WIN_OPTIONS = {
 }
 local _NOTHING_TO_SHOW = { "No info found." }
 
---- An individual infoview.
+--- An individual info.
+local Info = {}
+local next_info_id = 0
+
+--- A "view" on an info (i.e. window).
 local Infoview = {}
+local next_infoview_id = 0
 
---- Get the ID of the infoview corresponding to the current window.
+--- Get the ID of the current infoview.
 local function get_id()
-  return vim.api.nvim_win_get_tabpage(0)
+  return infoview._by_tabpage[vim.api.nvim_win_get_tabpage(0)]
 end
 
 --- Get the infoview corresponding to the current window.
 function infoview.get_current_infoview()
   return infoview._by_id[get_id()]
 end
 
+--- Get the info corresponding to the current window.
+function infoview.get_current_info()
+  return infoview._info_by_id[infoview.get_current_infoview().info]
```

Comment:
Should just be able to make `Infoview.info` be the info and then this function isn't needed if I follow.